static/src/javascripts/projects/common/modules/commercial/braze/brazeBanner.tsx

```diff
@@ -216,29 +238,32 @@ const canShow = async () => {
 	}
 
 	// Don't load Braze SDK for paid content
-	if (config.get('page').isPaidContent) {
+	if (config.get('page.isPaidContent')) {
 		return false;
 	}
 
 	try {
-		const result = await getMessageFromBraze(apiKey, brazeUuid);
-		const timeTaken = bannerTiming.end();
+		message = await getMessageFromBraze(apiKey, brazeUuid);
```

Comment:
`message` has type `BrazeMessageInterface`, and `getMessageFromBraze` returns a `BrazeMessage`. Are these two types the same?

Reply:
This is a good observation, they are effectively the same thing.

`BrazeMessage` is the actual class which we wrap messages returned from the Braze SDK in. It's defined here: https://github.com/guardian/braze-components/blob/main/src/logic/BrazeMessages.ts#L27

I defined this `BrazeMessageInterface` to represent the public interface supplied by instances of that class. In the message forcing logic it's hard to return a `BrazeMessage` from the forced json. TypeScript complains about the constructor interface for example. But maybe a better approach here would be to change the code in `@guardian/braze-components` to return a `BrazeMessageInterface` (which an instance of `BrazeMessage` would conform to) and at least the type definitions would all live in one centralised place. I might still vote to go with what we've got in this PR as an intermediate step.

Reply:
That makes sense, perhaps it's enough to add a comment above `BrazeMessageInterface`